pages/markets/[marketid].tsx

```diff
@@ -161,46 +162,153 @@ const Market: NextPage<{
-              value={`${prizePool} ${store?.config.tokenSymbol}`}
-            />
+            }).format(createdAt)}
+          </HeaderStat>
+          <HeaderStat label={hasDatePassed(ends) ? "Ended" : "Ends"}>
+            {new Intl.DateTimeFormat("en-US", {
+              dateStyle: "medium",
+            }).format(ends)}
+          </HeaderStat>
+          {token ? (
+            <HeaderStat label="Volume">
+              {/* TODO: replace num formatting with util function */}
+              {new Intl.NumberFormat("default", {
+                maximumSignificantDigits: 3,
+                notation: "compact",
+              }).format(volume)}
+              &nbsp;
+              {token}
+            </HeaderStat>
           ) : (
-            <></>
+            <Skeleton width="150px" height="24px" />
           )}
-          {pool?.liquidity != null ? (
-            <LiquidityPill
-              liquidity={Number.isNaN(pool.liquidity) ? 0 : pool.liquidity}
-            />
+          {prizePool >= 0 && token ? (
+            <HeaderStat label="Prize Pool">
+              {/* TODO: replace num formatting with util function */}
+              {new Intl.NumberFormat("default", {
+                maximumSignificantDigits: 3,
+                notation: "compact",
+              }).format(prizePool)}
+              &nbsp;
+              {token}
+            </HeaderStat>
           ) : (
-            <></>
+            <Skeleton width="150px" height="24px" />
           )}
+          {subsidy >= 0 && token ? (
+            <HeaderStat label="Subsidy" border={false}>
+              {/* TODO: replace num formatting with util function */}
+              {new Intl.NumberFormat("default", {
+                maximumSignificantDigits: 3,
+                notation: "compact",
+              }).format(subsidy)}
+              &nbsp;
+              {token}
+            </HeaderStat>
+          ) : (
+            <Skeleton width="150px" height="24px" />
+          )}
+        </div>
+        <div className="flex flex-wrap justify-center gap-2.5">
+          <Tag className={`${status === "Active" && "!bg-green-lighter"}`}>
+            {status === "Active" && (
+              <span className="text-green">&#x2713; </span>
+            )}
+            {status}
+          </Tag>
+          {tags?.map((tag, index) => {
+            return <Tag key={index}>{tag}</Tag>;
+          })}
         </div>
+      </header>
+    );
+  };
+
+  //data for MarketHeader
+  const token = store?.config?.tokenSymbol && store.config.tokenSymbol;
+  const createdAt = indexedMarket?.pool?.createdAt
+    ? new Date(indexedMarket.pool.createdAt).getTime()
+    : Number(indexedMarket.period.start);
+  const ends = Number(indexedMarket.period.end);
+  const volume = indexedMarket?.pool?.volume
+    ? Number(indexedMarket?.pool?.volume)
+    : 0;
+  // -1 to indicate loading state since type is number
+  const prize = prizePool === undefined ? -1 : prizePool ? prizePool : 0;
```

Comment:
Can't we use the `undefined` check to see if it's loading rather than setting another value?